biztalk/core/step-2-for-azure-create-an-edi-agreement.md

```diff
@@ -20,7 +20,7 @@ In this topic, you will create partners using the Azure BizTalk portal available
 
 ### To create partners
 
-1.  Using your Microsoft account, log into the portal at [https://go.microsoft.com/fwlink/p/?LinkId=235056](https://go.microsoft.com/fwlink/p/?LinkId=235056).
+1.  Using your Microsoft account, log into the portal.
```

Comment:
This doc is so outdated, I think when they say the "Azure BizTalk portal", they mean "Azure portal" and are referring to "Azure Logic Apps", which used to be called "BizTalk Services". Oh dear.

```suggestion
1. Sign in to the [Azure portal](https://portal.azure.com) with your Microsoft account.
```